scripts/prepare_sector_network.py

```diff
@@ -2347,17 +2348,394 @@ def add_biomass(n, costs):
+            - costs.at["solid biomass", "CO2 intensity"],
+            efficiency2=costs.at["solid biomass", "CO2 intensity"]
+            * costs.at["biomass CHP capture", "capture_rate"],
+            lifetime=costs.at["solid biomass boiler steam CC", "lifetime"],
+        )
+
+    if options["industry_t"]["low_T"]["methane"]:
+        n.madd(
+            "Link",
+            nodes,
+            suffix=" gas for lowT industry",
+            bus0=spatial.gas.nodes,
+            bus1=nodes + " lowT industry",
+            bus2="co2 atmosphere",
+            carrier="lowT industry methane",
+            p_nom_extendable=True,
+            p_min_pu=must_run,
+            efficiency=costs.at["gas boiler steam", "efficiency"],
+            capital_cost=costs.at["gas boiler steam", "fixed"]
+            * costs.at["gas boiler steam", "efficiency"],
+            marginal_cost=costs.at["gas boiler steam", "VOM"],
+            efficiency2=costs.at["gas", "CO2 intensity"],
+        )
+
+        eta = (
+            costs.at["gas boiler steam", "efficiency"]
+            - costs.at["gas", "CO2 intensity"]
+            * costs.at["biomass CHP capture", "heat-input"]
+        )
+        n.madd(
+            "Link",
+            nodes,
+            suffix=" gas for lowT industry CC",
+            bus0=spatial.gas.nodes,
+            bus1=nodes + " lowT industry",
+            bus2=spatial.co2.nodes,
+            bus3="co2 atmosphere",
+            carrier="lowT industry methane CC",
+            p_nom_extendable=True,
+            p_min_pu=must_run,
+            efficiency=eta,
+            capital_cost=costs.at["gas boiler steam", "fixed"]
+            * costs.at["gas boiler steam", "efficiency"]
+            + costs.at["biomass CHP capture", "fixed"]
+            * costs.at["gas", "CO2 intensity"],
+            marginal_cost=costs.at["gas boiler steam", "VOM"],
+            efficiency3=costs.at["gas", "CO2 intensity"]
+            * (1 - costs.at["biomass CHP capture", "capture_rate"]),
+            efficiency2=costs.at["gas", "CO2 intensity"]
+            * costs.at["biomass CHP capture", "capture_rate"],
+            lifetime=costs.at["gas boiler steam", "lifetime"],
+        )
+
+    if options["industry_t"]["low_T"]["heat_pumps"]:
+        eta = costs.at["industrial heat pump high temperature", "efficiency"]
+        n.madd(
+            "Link",
+            nodes,
+            suffix=" industrial heat pump steam for lowT industry",
+            bus0=nodes,
+            bus1=nodes + " lowT industry",
+            carrier="lowT industry heat pump",
+            p_nom_extendable=True,
+            p_min_pu=must_run,
+            efficiency=eta,
+            capital_cost=costs.at["industrial heat pump high temperature", "fixed"]
+            * eta,
+            marginal_cost=costs.at["industrial heat pump high temperature", "VOM"],
+            lifetime=costs.at["industrial heat pump high temperature", "lifetime"],
+        )
```

Comment:
Why do we take `industrial heat pump high temperature` instead of `industrial heat pump medium temperature` from technology data?

Reply:
Just a guess, but probably medium temperature heat pump can not provide heat for processes up to 500°C, but maybe check DEA documentation

Reply:
So I have just checked, high temperature in the Danish Energy Data means up to 150°C for heat pumps